doc/lightweight-styling.md

```diff
@@ -0,0 +1,58 @@
+# Lightweight Styling
+
+[Lightweight styling](https://learn.microsoft.com/en-us/windows/apps/design/style/xaml-styles#lightweight-styling) is a way to customize the appearance of XAML controls by **overriding** the system brushes that they use. Lightweight styling is applied in the Resources of the control itself:
+
+```xml
+<Button Content="Default Button Style" Style="{StaticResource MaterialFilledButtonStyle}" />
+
+<Button Content="Overridden Button Style" Style="{StaticResource MaterialFilledButtonStyle}" BorderThickness="2">
+	<Button.Resources>
+		<SolidColorBrush x:Key="FilledButtonForeground" Color="DarkGreen" />
+		<SolidColorBrush x:Key="FilledButtonBackground" Color="LightGreen" />
+		<SolidColorBrush x:Key="FilledButtonBorderBrush" Color="DarkGreen" />
+	</Button.Resources>
+</Button>
+```
+
+![Material - Button lightweight styling](assets/material-lightweight-styling-anatomy.png)
+
+Most controls have multiple states, such as **PointerOver** (mouse is hovered over), **Pressed** (control is pressed on), and **Disabled** (control is not interactable). These states are appended onto the endings of the resource keys: ButtonForeground*PointerOver*, ButtonForeground*Pressed*, and ButtonForeground*Disabled*. Some controls even have **Checked** and **Unchecked** states.
+
+```xml
+<Button Content="Overridden Button Style" Style="{StaticResource MaterialFilledButtonStyle}" BorderThickness="2">
+	<Button.Resources>
+		<SolidColorBrush x:Key="FilledButtonForeground" Color="DarkGreen" />
+		<SolidColorBrush x:Key="FilledButtonBackground" Color="LightGreen" />
+		<SolidColorBrush x:Key="FilledButtonBorderBrush" Color="DarkGreen" />
+
+		<!-- Overriding the PointerOver brushes -->
+		<SolidColorBrush x:Key="FilledButtonForegroundPointerOver" Color="DarkRed" />
+		<SolidColorBrush x:Key="FilledButtonBackgroundPointerOver" Color="LightPink" />
+		<SolidColorBrush x:Key="FilledButtonBorderBrushPointerOver" Color="DarkRed" />
+	</Button.Resources>
+</Button>
+```
+
+![Material - Button lightweight styling](assets/material-button-pointerover-lightweight-styling.png)
+
+## Resource Keys
+For more information about the lightweight styling resource keys used in each control, check out the following links:
```

Comment:
you can update the links now

Reply:
here and as well as in toc.yml

Reply:
@Xiaoy312 let me know if the toc.yml file was done right

also PipsPager is still being discussed but will add the link when thats done

Reply:
@Xiaoy312 were u referring to xref?

Reply:
re arajamatt:
toc and this file both look good
comment it out for now, it can be restored in the pipspaper related pr if it ever come to light

re weitzhandler:
just plain relative path

Reply:
@Xiaoy312, what's the reason we're not using UIDs and xref?